main.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+#include <string>
+#include "Bug.h"
+#define N 3
+
+using namespace std;
+
+class Backlog {
+public:
+
+    void OutPutDev(Bug List[], string Dev) {
+        cout << "======================================================" << endl;
+        for (int i = 0; i < N; i++) {
+            if (List[i].GetDeveloper() == Dev && List[i].GetStatus() == "RESOLVED")  {
+                OutPutObj(List[i]);
+            }
+        }
+        cout << "======================================================" << endl;
+    }
+
+    void SortBacklog(Bug List[]) {
```

Comment:
Please rename to make clear how you sort (like "bubble sort") and descending or ascending